cloud/endagaweb/forms/dashboard_forms.py

```diff
@@ -153,6 +153,19 @@ def __init__(self, *args, **kwargs):
 
 class ChangePasswordForm(PasswordChangeForm):
```

Comment:
Thanks,yes ,As django have four default validators  and they having their own policy text.
But as per our requirement we don't need UserAttributeSimilarityValidator,CommonPasswordValidator. so only NumericPasswordValidator and MinimumLengthValidator we could use.But NumericPasswordValidator only check numeric ,it doesn't check alphanumeric and they don't have validator check for special character.For that regexValidator need to use.
For all this , made custom validator custom_password_validators, which validated min_length,alphanumeric and special character. And our custom policy provided as help text.
Removed override of all those fields , added custom validators to display our policy and validate password strength.

commitId:1f9e597e4ffae62e4e54c4447c02bc4e7f3a1232
![defaultdjangovalidator](https://user-images.githubusercontent.com/26892520/26891862-1d4811e4-4bd4-11e7-9faf-c0754794d93b.png)